app/assets/javascripts/application.js

```diff
@@ -16,6 +16,9 @@
 //= require turbolinks
 //= require_tree .
 
+var lights = lights || []
+var isPartyOn = isPartyOn || false
```

Comment:
I can't see what good these do. These values won't be used in admin view, and in the lights view they are set in index.html.erb